source/main/modules/window-manager/index.ts

```diff
@@ -219,6 +223,36 @@ export default class WindowManager {
       return
     }
 
+
+
+    this._mainWindow.on('show', () => {
+      if (process.platform === 'win32' || process.platform === 'linux') {
+        
+        if (this._tray == null) {
+          this._tray = new Tray(path.join(__dirname, 'assets/icons/icon.ico'))
+    
```

Comment:
Fails `yarn lint` : ` 233:1   error    Trailing spaces not allowed  no-trailing-spaces`